### ISSUE TITLE
JLC assembly compatible BOM

### ISSUE THREAD

**Issue by allexoll** [probe-rs/rusty-probe#4]
having a BOM compatible with JLC assembly service would be pretty helpfull for accessibility.

Might have a look later if I find the time.

**Comment by korken89** [comment]
It would be awesome! I'd also like there to be such, ping me if you start looking and I can give feedback.
I think even if we need to update the design it would be worth it.

**Comment by allexoll** [comment]
#6 should be a start, but I have not ordered some using that ( no stock on any semiconductor anyway...)

**Comment by korken89** [comment]
Indeed, I have no idea how lcsc works so finding a suitable LDO should be the only thing that needs fixing right?

**Comment by allexoll** [comment]
no, what I meant is that when I ordered my small batch, they did not have any ic in stock, mostly due to shortage. From what i understand from JLC/LCSC, you can order the parts in advance for them to have them in stock when you assemble it. it just was not worth it for the 10 I built, and for those I only asked them to place the passive (Rs, Cs) and a few others: Osc, type-c connector, and so on. see photo: 
![IMG_6041](https://user-images.githubusercontent.com/261441/204244291-b6ca53c6-3c60-4209-816b-3b00bf703bbd.JPG)


I have assembled one with components I ordered from mouser, but I did not have time to test it yet.

**Comment by korken89** [comment]
Ahhh, I understand. Looks great!

FYI - I just pushed updates to the design so we can use the active discharge version of NCP133 (which seems to be easy to get).
It should not impact the code though, so this board should work fine in the end.

**Comment by korken89** [comment]
Also I added CI actions to generate gerbers, images of PCB etc for future use.

**Comment by korken89** [comment]
I gave this a look again, there seems to be a lot better part selection on JLCPCB now - I'm quite sure one can adapt the board into an JLCPCB friendly version.

**Comment by berkus** [comment]
We have a JLCPCB bom now, so I guess this can be closed?
